### PR TITLE
core: Removed external force switching flags

### PR DESCRIPTION
**Pull request by fweik**
Description of changes:
 - Removed flags to switch external force/torque on and
   of per particle. This does not actually improved performance
   and creates some noise.
- Not user visible.

### CODE REVIEW

File: src/core/particle_data.hpp [hunk 1]
```diff
@@ -62,20 +62,10 @@ enum {
 /** \ref ParticleProperties::ext_flag "ext_flag" value for particle subject to
  *  an external force
  */
-#define PARTICLE_EXT_FORCE 1
-/** \ref ParticleProperties::ext_flag "ext_flag" value for fixed coordinate
- *  coord. */
```
Comment (jngrad) [inline src/core/particle_data.hpp:67]: did you remove the correct docstring?

Comment (fweik) [reply src/core/particle_data.hpp:67]: No, I'll address that in a separate pull request.

**Review by jngrad — APPROVED** [review]
LGTM

**Comment by fweik** [comment]
bors r=jngrad

**Pull request merged**